commec/screeners/check_benign.py

```diff
@@ -124,8 +124,7 @@ def check_for_benign(query, coords, benign_desc):
         logging.info("\t...no Synbio sequence hits\n")
     else:
         blastn = readblast(blast)  # synbio parts
-        blastn = trimblast(blastn)
-        blastn = tophits(blastn)
+        blastn = get_top_hits(blastn)
```

Comment:
_trim_overlapping is still in import? Unused, delete? - I see its been folded into get_top_hits. Love the use of private _ modifier. I think there's some more functions around that could benefit from that.